src/features/filters/components/filter-builder.tsx

```diff
@@ -334,6 +336,18 @@ function FilterBuilderForm({
                       }
                       values={Array.isArray(filter.value) ? filter.value : []}
                     />
+                  ) : filter.type === "arrayOptions" ? (
```

Comment:
this is the same as above, right? Could we just write `filter.type === "stringOptions" || filter.type === "arrayOptions"` above?